src/middleware/auth/base-auth.ts

```diff
@@ -59,24 +46,25 @@ export abstract class AbstractAuthHandler implements IAuthResourceHandler
         if (rule === null) {
             return this.returnError(500, `Internal error. Issue with finding the rule for the path ${request.path}`)
         } else {
-            // Tries to get token from the request and other preps
-            const _setup = this.setupAuthToken(request)
-
-            // If token is not provided - try to proceed without it
-            // If token is provided - verify it and get the scopes
-            if (_setup.status !== 200) {
-                return _setup 
+            // If the user is not authenticated - return error
+            if (!request.user.isAuthenticated) {
+                return this.returnError(400, "Unauthorized error: Seems like you are not authenticated. Please follow the authentication process using 'LogIn' button")
             }
-            const resourceAPI = AbstractAuthHandler.buildResourceAPIUrl(request)
-            if (!resourceAPI) {
-                return this.returnError(500, `Internal error. Issue with building resource API for the path ${request.path}`)
+            // Tries to get customerId from the logTo user structure
+            if (request.user && request.user.claims) {
+                this.customerId = request.user.claims.sub
+            } else {
+                return this.returnError(500, "Unauthorized error: Seems like authentication process was corrupted and there are problems with getting customerId")
```

Comment:
500 is the wrong error code for this type: either this is not an "Unauthorised" case, or the error code needs to change.

Reply:
So, here it would be a real 500 error (maybe text should be changed) cause here I assume that LogTo middleware added `user` property for the `request` object. Such state is not under the client control